lib/logidze/versioned_association.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+require 'active_support'
+
+module Logidze
+  module VersionedAssociation
+    module SingularAssociation
+      def reader(force_reload = false)
```

Comment:
> Issue gone wild, didn't it? :)

Oh, yeah)

`stale_target?` is a standard AR mechanism to check, whether we need reload or not, and it's used all over the associations codebase. Thus we're unlikely to miss something. And we should not care about `#reload` ourselves. I hope so, at least.

> All double at! wouldn't affect association ...

Ok.